components/Internship.js

```diff
@@ -32,8 +30,8 @@ function Internship({
             />
           </a>
         </div>
-        <div className="properties__caption">
-          <div className="courseInfo">
+        <div className={styles.internshipInfo}>
+          <div className={styles.courseInfo}>
```

Comment:
Some CSS aren't written.

Reply:
ok! can you attach the ss too please

Reply:
<img width="926" alt="Screenshot 2023-06-05 at 22 35 23" src="https://github.com/anmode/grabtern-frontend/assets/91741581/a2611f52-b738-4407-a089-a1434011bf44">
<img width="1070" alt="Screenshot 2023-06-05 at 22 34 16" src="https://github.com/anmode/grabtern-frontend/assets/91741581/195bb360-f7e5-45bd-9a43-4c96e269366f">

Since, there are no css for the above, I made the changes as it was earlier. 
Is it alright or can I remove these classes too?

Reply:
remove the class whole properties are not written.

Reply:
I found that the CSS of internship.js got distorted when I removed these css classes.
Therefore, in some places even I added css properties also so that the changes remains alright.